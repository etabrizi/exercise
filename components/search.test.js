import * as React from 'react';
import Search from './search';
import renderer from 'react-test-renderer';

const Props = {
    excercises: [{}, {}, {}],
    searchTerm: 'pull ups',
    updateCard: () => { },
    getSearchValue: () => { },
    searchToggle: true
}

describe('Search component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Search {...Props} />)
            .toJSON();
        expect(tree).toMatchSnapshot()
    });
});   
