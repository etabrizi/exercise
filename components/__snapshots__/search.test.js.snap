// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders correctly 1`] = `
<div
  className="jsx-583777132 open"
>
  <input
    className="jsx-583777132"
    onChange={[Function]}
    placeholder="Search for your workout . . . "
    type="search"
  />
</div>
`;
